setup.py

```diff
@@ -13,6 +13,7 @@
     "copier==5.1.0",
     "dbt>=0.21, <0.22",
     "fsspec",
+    "dbt-airflow-manifest-parser>=0.14",
```

Comment:
chyba bezpieczniej jest fixować wersję

Reply:
nb: a przynajmniej do minora (choć `0.x.y` nie zakłada wstecznej kompatybilności to chyba fajnie ją trzymać chociażby dla bugfixów)